doc/build.md

```diff
@@ -23,22 +23,26 @@ Before building the project, consult the following table to ensure you use the r
 
 #### 2. Download and Build Pixar USD 
 
-See Pixar's official github page for instructions on how to build USD: https://github.com/PixarAnimationStudios/USD
+See Pixar's official github page for instructions on how to build USD: https://github.com/PixarAnimationStudios/USD . Pixar has recently removed support for building Maya USD libraries/plug-ins in their github repository and ```build_usd.py```. When building the maya-usd project, it is important the recommended ```Pixar USD``` and ```Animal Logic``` commitID or tag from the table below are used: 
 
-For additional information on building USD, see the ***Building Pixar USD*** section below.
+|   Required    |      ![](images/pxr.png)          |         ![](images/al.png)            |
+|:------------: |:---------------:  |:-----------------:    |
+| CommitID/tag  | ceca736(>19.11)   | 631a2911(>0.34.0)     |
+
+
+For additional information on building Pixar USD, see the ***Additional Build Instruction*** section below.
 
 ***NOTE:*** Make sure that you don't have older USD locations in your ```PATH``` and ```PYTHONPATH``` environment settings. ```PATH``` and ```PYTHONPATH``` are automatically adjusted inside the project to point to the correct USD location. See ```cmake/usd.cmake```.
 
-#### 3. Download Universal Front End (UFE) 
+#### 3. Universal Front End (UFE)
 
-The Universal Front End (UFE) is a DCC-agnostic component that allows Maya to browse and edit data in multiple data models. This allows Maya to edit pipeline data such as USD.
-UFE is developed as a separate binary component, and is therefore versioned separately from Maya. UFE is installed as a built-in component of Maya.
+The Universal Front End (UFE) is a DCC-agnostic component that allows Maya to browse and edit data in multiple data models. This allows Maya to edit pipeline data such as USD. UFE comes installed as a built-in component with Maya 2019 and later. UFE is developed as a separate binary component, and therefore versioned separately from Maya. Current shipped version of UFE is **1.0.0**
 
-UFE headers and Doxygen documentation is shipped with Maya Devkit and can be downloaded from the link below: 
+To build the project with UFE support, you will need to use the headers and libraries included in the ***Maya Devkit***:
 
-https://www.autodesk.com/developer-network/platform-technologies/maya  
```

Comment:
@BigRoy I changed the wording a little bit and hopefully it makes more sense?

Reply:
Definitely clearer to me! Lgtm.